modules/olm-rukpak-bd.adoc

```diff
@@ -6,13 +6,13 @@
 [id="olm-rukpak-bd_{context}"]
 = BundleDeployment
 
+In {product-title} 4.16, the RukPak `BundleDeployment` API can include an embedded spec for a desired bundle and indicates that it should be active. This includes pivoting from older versions of an active bundle.
+
 [WARNING]
 ====
 A `BundleDeployment` object changes the state of a Kubernetes cluster by installing and removing objects. It is important to verify and trust the content that is being installed and limit access, by using RBAC, to the `BundleDeployment` API to only those who require those permissions.
 ====
 
-The RukPak `BundleDeployment` API points to a `Bundle` object and indicates that it should be active. This includes pivoting from older versions of an active bundle. A `BundleDeployment` object might also include an embedded spec for a desired bundle.
-
 Much like pods generate instances of container images, a bundle deployment generates a deployed version of a bundle. A bundle deployment can be seen as a generalization of the pod concept.
```

Comment:
@ Eng - This change is an assumption, attempting to reconcile how it works now for lack of a `Bundle` object (based on the other context described nearby), but please correct.